apps/sms/views/shared/js/startup.js

```diff
@@ -94,7 +97,20 @@ var Startup = exports.Startup = {
       // Init UI Managers
       TimeHeaders.init();
       ConversationView.init();
-      MessagingClient.init(App.instanceId);
+
+      // Fetch mmsSizeLimitation and max concat
+      Settings.init();
+
+      var serviceEndpoint;
+      const servicePath = '/services/js/messaging/messaging_service.js';
+      if (Settings.useSharedWorkerServices) {
+        serviceEndpoint = new SharedWorker(servicePath);
+      } else {
+        serviceEndpoint = exports;
+        LazyLoader.load(servicePath).then(() => MessagingService.init());
```

Comment:
We also can lazy load service file all the time and then make ```delete window.MessagingService``` as well, I don't like both methods, so didn't bother much :)